src/classes/Bot.ts

```diff
@@ -187,14 +191,25 @@ export = class Bot {
         return this.trades.checkEscrow(offer);
     }
 
-    messageAdmins(type: string, message: string): void {
-        if (!this.alertTypes.includes(type)) {
+    messageAdmins(message: string, exclude: string[] | SteamID[]): void;
+
+    messageAdmins(type: string, message: string, exclude: string[] | SteamID[]);
+
+    messageAdmins(...args): void {
+        const type: string | null = args.length === 2 ? null : args[0];
+
+        if (type !== null && !this.alertTypes.includes(type)) {
             return;
         }
 
-        this.admins.forEach(steamID => {
-            this.sendMessage(steamID, message);
-        });
+        const message: string = args.length === 2 ? args[0] : args[1];
+        const exclude: string[] = (args.length === 2 ? [] : args[2]).map(steamid => steamid.toString());
+
+        this.admins
+            .filter(steamID => exclude.includes(steamID.toString()))
+            .forEach(steamID => {
+                this.sendMessage(steamID, message);
+            });
     }
```

Comment:
I think `messageAdmins` is corrupted.
[messageAdmins-not-working](https://user-images.githubusercontent.com/47635037/78546670-62d34e00-7830-11ea-9d1a-556dfdfcdfee.PNG)